code/modules/mob/living/carbon/human/life.dm

```diff
@@ -610,6 +610,19 @@
 			if(prob(nausea))
 				delayed_vomit()
 
+		if(CE_ITCH in chem_effects)
+			var/itching = chem_effects[CE_NOITCH]
+			if(CE_ITCH in chem_effects)
+				itching -= chem_effects[CE_NOITCH]
```

Comment:
Why are you checking itching twice?

Reply:
Whoops. Mixed up `NOITCH` and `ITCH`. Should be good now.